reference/6/Microsoft.PowerShell.Core/About/about_CimSession.md

```diff
@@ -0,0 +1,65 @@
+---
+ms.date:  04/28/2018
+schema:  2.0.0
+locale:  en-us
+keywords:  powershell,cmdlet
+title:  about_CimSession
+---
+
+# About CimSession
+
+## SHORT DESCRIPTION
+
+Describes a CimSession object and the difference between CIM sessions and
+Windows PowerShell sessions.
+
+## LONG DESCRIPTION
+
+A Common Information Model (CIM) session is a client-side object that
+represents a connection to a local computer or a remote computer. You can
+use CIM sessions as an alternative to Windows PowerShell sessions
+(PSSessions). Both approaches have advantages.
+
+You can use the [New-CimSession](../../CimCmdlets/New-CimSession.md) cmdlet to create a CIM session that contains
+information about a connection, such as computer name, the protocol used
+for the connection, session ID, and instance ID.
+
+After you create a CimSession object that specifies information required to
+establish a connection, Windows PowerShell does not establish the
+connection immediately. When a cmdlet uses the CIM session,
+Windows PowerShell connects to the specified computer, and then, when the
+cmdlet finishes, Windows PowerShell terminates the connection.
+
+If you create a PSSession instead of using a CIM session,
+Windows PowerShell validates connection settings, and then establishes and
```

Comment:
Unless something is "Windows PowerShell" specific, we should just use "PowerShell" (applies throughout the doc).